src/blas/backends/cublas/cublas_level1.cpp

```diff
@@ -56,6 +56,7 @@ inline void asum(Func func, cl::sycl::queue &queue, int64_t n, cl::sycl::buffer<
             cublasStatus_t err;
             // ASUM does not support negative index
             CUBLAS_ERROR_FUNC(func, err, handle, n, x_, std::abs(incx), res_);
+            cublasSetPointerMode(handle, CUBLAS_POINTER_MODE_HOST);
```

Comment:
Could you please add a brief description on the purpose of this change? (similar comment for the other ones as well)